packages/admin-portal/src/selector/app-detail.ts

```diff
@@ -1,18 +1,10 @@
 import { ReduxState } from '@/types/core'
-import { AppDetailModel } from '@reapit/foundations-ts-definitions'
+import { AppsState } from '@/reducers/apps'
 
-export const selectAppDetailState = (state: ReduxState) => {
-  return state?.appDetail
+export const selectAppDetailState = (state: ReduxState): AppsState['detail'] => {
```

Comment:
I think this should be AppModel instead of `AppsState['detail']`

Reply:
Agreed, see my comment above